Documentation/Index.rst

```diff
@@ -1,11 +1,36 @@
 .. include:: /Includes.rst.txt
+
 .. index:: ! Writing documentation
-.. _start:
 
 =====================
 Writing documentation
 =====================
 
+:Version:
```

Comment:
I would put this part below the cards with the deep links. All information can be gained from the left as well and the cards are necessary for a quick navigation, while the other links will be seldomly clicked. Lets not lose the advantage we just gained here.

Reply:
Sure. Adjust to your likes.

Reply:
It looks currently like this.
![Screen Shot 2022-03-22 at 09 56 15](https://user-images.githubusercontent.com/20297232/159443652-a33b034f-dd07-43dc-aa2f-9d1a2126bd55.png)